nano/secure/ledger.cpp

```diff
@@ -1203,6 +1203,26 @@ bool nano::ledger::block_not_confirmed_or_not_exists (nano::block const & block_
 	return result;
 }
 
+std::vector<std::pair<nano::block_hash, nano::block_hash>> nano::ledger::unconfirmed_frontiers () const
+{
+	std::vector<std::pair<nano::block_hash, nano::block_hash>> unconfirmed_confirmed_frontiers;
+	auto transaction (store.tx_begin_read ());
+	auto conf_height_i = store.confirmation_height_begin (transaction);
+
+	for (auto i (store.latest_begin (transaction)), n (store.latest_end ()); i != n; ++i, ++conf_height_i)
+	{
+		// Make sure the accounts match
+		debug_assert (conf_height_i->first == i->first);
+		if (i->second.block_count != conf_height_i->second.height)
+		{
+			auto frontier = i->second.head.to_string ();
```

Comment:
Why to_string?

Reply:
Probably artefact from earlier iteration of PR. Doesn't make sense to convert from hash to string back to hash 👎 Good spot!